spec/helpers/application_helper_spec.rb

```diff
@@ -379,4 +379,25 @@
       end
     end
   end
+
+  describe "#byline" do
+    let(:user1) { create(:user, login: "Beetle") }
+    let(:user2) { create(:user, login: "Muppet") }
+    let(:work) { create(:work, authors: [user1.default_pseud, user2.default_pseud]) }
+
+    before do
+      create(:locale, iso: "new")
+      I18n.backend.store_translations(:new, { support: { array: { words_connector: "，" } } })
+    end
+
+    it "is locale independent" do
```

Comment:
It's not super clear to me how, `helper.byline` still depends on `words_connector`.

Reply:
Perhaps a bad way to word it? The cache is locale independent, the resulting text is not and changes based on locale despite the cache staying the same